support/catalogs/images.go

```diff
@@ -110,31 +110,45 @@ func imageExistsFn(ctx context.Context, hcp *hyperv1.HostedControlPlane, pullSec
 		if err == nil {
 			return true, nil
 		}
-		if strings.Contains(err.Error(), "manifest unknown") {
+		if strings.Contains(err.Error(), "manifest unknown") || strings.Contains(err.Error(), "access to the requested resource is not authorized") {
```

Comment:
this probably deserves a comment on why this errors are expectional?

Reply:
This new error was the main cause of the initial issue, for some reason the RH registry started to trigger it, even if the auth was correct... I've added both errors to the check, in order to move on and don't block the AWS instance generation.

Reply:
My ask is that this should be articulated in a comment in code, so any reader can understand the rationale.